src/vs/editor/contrib/referenceSearch/browser/referencesWidget.ts

```diff
@@ -479,12 +496,19 @@ export interface SelectionEvent {
 }
 
 /**
+ * Map of FileReferences by id.
+ */
+type FileReferenceSet = Map<string, FileReferences>;
```

Comment:
Artifact of me attempting to use Set<FileReferences> and realizing that ES6 Sets are useless.

`{"foo": true} === {"foo": true} // false` and there is no way to define custom equality for objects so I had to use a Map here.